source/Ship.h

```diff
@@ -310,6 +310,8 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	int Crew() const;
 	int RequiredCrew() const;
 	void AddCrew(int count);
+	// Get this ship's crew equivalent value.
+	int CrewValue() const;
```

Comment:
Still have to shuffle this one up. The comment could also use some more description.
```suggestion
	// Get this ship's crew value, which is the max between its current crew or
	// required crew plus the value of the "crew equivalent" attribute.
	int CrewValue() const;
	void AddCrew(int count);
```

Reply:
I moved crewvalue down in the cpp
sure if will add more comments caus you want me to but looking at the cpp function is pretty straightforward isn't it ?

Reply:
> I moved crewvalue down in the cpp

Oh, I thought you had moved it below RequiredCrew.

> looking at the cpp function is pretty straightforward isn't it

I suppose so.